app/models/miq_request.rb

```diff
@@ -147,9 +147,7 @@ def call_automate_event(event_name)
     MiqAeEvent.raise_evm_event(event_name, self, build_request_event(event_name))
     _log.info("Raised  event [#{event_name}] to Automate")
   rescue MiqAeException::Error => err
-    message = _("Error returned from %{name} event processing in Automate: %{error_message}") %
-                {:name => event_name, :error_message => err.message}
-    raise
+    raise err, _("Error returned from #{event_name} event processing in Automate: #{err.message}")
```

Comment:
Yep. I wonder if rubocop or miq-bot could be taught to complain about `_("#{dstr}", ..)`.

Or maybe just have it do the extract-strings-for-translation thing, and report the diff?